test/compilable/b19442.d

```diff
@@ -0,0 +1,8 @@
+// https://issues.dlang.org/show_bug.cgi?id=19442
+enum x1 = 42;
+enum x2 = mixin('x', 1);
+enum x3 = mixin(wchar('x'), 2);
+enum x4 = mixin(dchar('x'), 3);
```

Comment:
Instead of explicitly using `wchar` and `dchar` I suggest using characters that require `wchar` and `dchar`. For example, for `wchar`: `'ö'` and for `dchar`: `'🍺'`.

Reply:
yeah, i'll try, assuming they are part of the "universal identifier char set" (it's the obscure rule D follows, layered on C). Not sure for the chop.

Reply:
maybe it's rather called "universal alphas"

Reply:
You can mix it into a string, where there's no limitations of what can be stored.

Reply:
okay done